### PR TITLE
Make the OSSL_trace manual conform with man-pages(7)

### PR DESCRIPTION
**Pull request by levitte**
Details from man-pages(7) that are used:

    Formatting conventions for manual pages describing functions

        ...
        Variable names should, like argument names, be specified in italics.
        ...

    Formatting conventions (general)

        ...
        Special macros, which are usually in uppercase, are in bold.
        Exception: don't boldface NULL.
        ...

### CODE REVIEW

File: doc/man3/OSSL_trace_set_channel.pod [hunk 1]
```diff
@@ -128,59 +128,59 @@ prematurely.
...
 Traces the ENGINE algorithm table selection.
 
 More precisely, engine_table_select(), the function that is used by
 RSA, DSA (etc) code to select registered ENGINEs, cache defaults and
 functional references (etc), will generate trace summaries.
 
-=item C<OSSL_TRACE_CATEGORY_ENGINE_REF_COUNT>
+=item B<OSSL_TRACE_CATEGORY_ENGINE_REF_COUNT>
 
 Tracds the ENGINE reference counting.
```
Comment (t8m) [inline doc/man3/OSSL_trace_set_channel.pod:149]: when you're at it, can you please fix this `Tracds` typo?

**Review by t8m — APPROVED** [review]
Approved with or without the Tracds typo fix.

File: doc/man3/OSSL_trace_set_channel.pod (hunk 1 above)
Comment (levitte) [reply doc/man3/OSSL_trace_set_channel.pod:149]: Sure

**Comment by levitte** [comment]
Merged

0f2380066de6436c0e8debfad1391db134ad4c25 Make the OSSL_trace manual conform with man-pages(7)

**Pull request closed without merging**